config/src/config/secure_backend_config.rs

```diff
@@ -164,6 +168,43 @@ impl OnDiskStorageConfig {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct RocksDbStorageConfig {
+    // Required path for on disk storage
+    pub path: PathBuf,
+    /// A namespace is an optional portion of the path to a key stored within OnDiskStorage. For
+    /// example, a key, S, without a namespace would be available in S, with a namespace, N, it
+    /// would be in N/S.
+    pub namespace: Option<String>,
+    #[serde(skip)]
+    data_dir: PathBuf,
```

Comment:
why this skips serde?

Reply:
Not entirely sure. I've just copied the OnDiskStorageConfig (which does the same thing). Didn't have time to look into why it skips it 🤔